### PR TITLE
[CHORE]: cleanup

### PR DESCRIPTION
**Pull request by ashutosh887**
### Changes proposed
removed unwanted dependencies: 
* lottie-react
* react-drag-drop-files
* react-particles

### CODE REVIEW

**Review by Yazdun — APPROVED** [review]
Thanks Ashutosh! I'm not sure about removing the lottie-react since we may want to use it in the future, but I'm going to approve this for now. @rohitdasu what do you think about removing lotties?

**Comment by ashutosh887** [comment]
@Yazdun as discussed in #270 

These were the unwanted dependencies for now: [here](https://github.com/rohitdasu/projectmate/issues/270#issuecomment-1528760190)

**Comment by rohitdasu** [comment]
> @Yazdun as discussed in #270
> 
> These were the unwanted dependencies for now: [here](https://github.com/rohitdasu/projectmate/issues/270#issuecomment-1528760190)

hi @ashutosh887 i agree with @Yazdun about the lottie package. we may need this package in future. could you make sure it stays. thanks

File: package.json [hunk 1]
```diff
@@ -23,21 +23,18 @@
     "flowbite": "^1.6.3",
     "flowbite-react": "^0.3.8",
     "framer-motion": "^9.1.7",
-    "lottie-react": "^2.4.0",
     "moment": "^2.29.4",
     "next": "^13.2.1",
     "next-auth": "^4.19.2",
     "next-themes": "^0.2.1",
```
Comment (rohitdasu) [inline package.json:29]: next themes can be removed

File: package.json [hunk 2]
```diff
@@ -23,21 +23,18 @@
...
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-drag-drop-files": "^2.3.8",
     "react-error-boundary": "^3.1.4",
     "react-hook-form": "^7.43.9",
     "react-hot-toast": "^2.4.0",
     "react-icons": "^4.7.1",
     "react-markdown": "^8.0.5",
-    "react-particles": "^2.8.0",
     "react-quill": "^2.0.0",
```
Comment (rohitdasu) [inline package.json:38]: react-quill also not needed.

**Comment by ashutosh887** [comment]
Great I'll remove them

**Comment by ashutosh887** [comment]
@rohitdasu updated as per your call

Should I replace img tag here?
https://github.com/rohitdasu/projectmate/blob/5fa748431cdf9755ff21dc93ce905df74fe98997/components/ProductHunt/ProductHunt.tsx#L18

and use next/image

PS: where does this component get rendered?

**Comment by rohitdasu** [comment]
> @rohitdasu updated as per your call
> 
> 
> 
> Should I replace img tag here?
> 
> https://github.com/rohitdasu/projectmate/blob/5fa748431cdf9755ff21dc93ce905df74fe98997/components/ProductHunt/ProductHunt.tsx#L18
> 
> 
> 
> and use next/image
> 
> 
> 
> PS: where does this component get rendered?

This is currently not in use. Can be kept as is. Later we will see where we can show this. Thanks

**Comment by rohitdasu** [comment]
@ashutosh887 thanks for the contribution. If possible please post the achievement in your twitter account tagging us (https://twitter.com/projectmateHQ)

**Comment by ashutosh887** [comment]
Sure @rohitdasu

**Pull request merged**